hw4_Zhurylo/js/index.js

```diff
@@ -0,0 +1,14 @@
+const firstNumber = +prompt('write first number')
```

Comment:
This is okay to use `+` for type change. But I guess `Number()` is more understandable.